go/types/opcache.go

```diff
@@ -27,25 +45,65 @@ func newOpCache(vrw ValueReadWriter) *opCache {
 		WriteBuffer:            1 << 27, // 128MiB
 	})
 	d.Chk.NoError(err, "opening put cache in %s", dir)
-	return &opCache{ops: db, dbDir: dir, vrw: vrw}
+	return &opCacheDb{ops: db, dbDir: dir}
+}
+
+func newOpCache(vrw ValueReadWriter) *opCache {
+	incrementActiveColCount()
+	prefix := [prefixByteSize]byte{}
+	colId := atomic.AddUint32(&collectionId, 1)
+	binary.LittleEndian.PutUint32(prefix[:], colId)
+	return &opCache{vrw: vrw, colId: colId, prefix: prefix[:]}
 }
 
 type opCache struct {
-	ops           *leveldb.DB
-	dbDir         string
-	vrw           ValueReadWriter
-	ldbKeyScratch [1 + hash.ByteLen]byte
-	keyScratch    [initialBufferSize]byte
-	valScratch    [initialBufferSize]byte
+	vrw              ValueReadWriter
+	ldbKeyScratch    [1 + hash.ByteLen]byte
+	keyScratch       [initialBufferSize]byte
+	prefixKeyScratch [initialBufferSize]byte
+	valScratch       [initialBufferSize]byte
+	colId            uint32
+	prefix           []byte
 }
 
 type opCacheIterator struct {
+	opc  *opCache
```

Comment:
I don't see anywhere in the `opCacheIterator` methods that this gets used. Leftover from a previous version of the patch?